app/api/resources/metadata.py

```diff
@@ -259,3 +260,30 @@ async def get_genome_by_keyword(request: Request, assembly_accession_id: str):
     except Exception as ex:
         logging.error(ex)
         return response_error_handler({"status": 500})
+
+@router.get("/genome/{genome_uuid}/vep/file_paths")
+async def get_vep_file_paths(
+    request: Request,
+    genome_uuid: str,
+):
+    try:
+        vep_file_paths = MessageToDict(
+            grpc_client.get_vep_file_paths(genome_uuid=genome_uuid)
+        )
+        if len(vep_file_paths) == 0:
+            return responses.JSONResponse(
+                {
+                    "message": f"Could not find VEP file paths for genome {genome_uuid}."
+                },
+                status_code=404,
+            )
+
+        vep_file_paths_object = VepFilePaths(**vep_file_paths)
+        response_data = responses.JSONResponse(
```

Comment:
You can avoid using response_data as a holder variable and return `responses`  just like line 274 of this file.